apps/root/src/pages/earn/components/all-strategies-table/components/filters/index.tsx

```diff
@@ -0,0 +1,325 @@
+    };
+
+    const farmsFilter: FilterControl<string> = {
+      options: strategiesParameters.farms.map((farm) => ({
+        value: farm.id,
+        label: <Typography variant="bodySmallSemibold">{farm.name}</Typography>,
+        searchParams: [farm.name],
+      })),
+      filteredOptions: strategiesFilters.farms,
+      summaryLabel: intl.formatMessage(
+        defineMessage({
+          defaultMessage: 'Protocol',
+          description: 'all-strategies-table.filters.protocol',
+        })
+      ),
+      handleFilterChange: (filter) => dispatch(setFarmFilter(filter)),
+    };
+
+    const networksFilter: FilterControl<number> = {
+      options: strategiesParameters.networks.map((network) => ({
+        value: network.chainId,
+        label: <Typography variant="bodySmallSemibold">{network.name}</Typography>,
+        searchParams: [network.name, network.chainId.toString()],
+      })),
+      filteredOptions: strategiesFilters.networks,
+      summaryLabel: intl.formatMessage(
+        defineMessage({
+          defaultMessage: 'Network',
+          description: 'all-strategies-table.filters.network',
+        })
+      ),
+      handleFilterChange: (filter) => dispatch(setNetworkFilter(filter)),
+    };
+
+    const yieldTypesFilter: FilterControl<StrategyYieldType> = {
+      options: strategiesParameters.yieldTypes.map((yieldType) => ({
+        value: yieldType.value,
+        label: <Typography variant="bodySmallSemibold">{yieldType.label}</Typography>,
+        searchParams: [yieldType.label],
+      })),
+      filteredOptions: strategiesFilters.yieldTypes,
+      summaryLabel: intl.formatMessage(
+        defineMessage({
+          defaultMessage: 'Yield Type',
+          description: 'all-strategies-table.filters.yield-type',
+        })
+      ),
+      handleFilterChange: (filter) => dispatch(setYieldTypeFilter(filter)),
+      hideSearch: true,
+    };
+
+    const guardiansFilter: FilterControl<string> = {
+      options: strategiesParameters.guardians.map((guardian) => ({
+        value: guardian.id,
+        label: <Typography variant="bodySmallSemibold">{guardian.name}</Typography>,
+        searchParams: [guardian.name],
+      })),
+      filteredOptions: strategiesFilters.guardians,
+      summaryLabel: intl.formatMessage(
+        defineMessage({
+          defaultMessage: 'Guardians',
+          description: 'all-strategies-table.filters.guardians',
+        })
+      ),
+      handleFilterChange: (filter) => dispatch(setGuardianFilter(filter)),
+    };
+
+    return [networksFilter, assetFilter, rewardsFilter, farmsFilter, yieldTypesFilter, guardiansFilter];
+  }, [intl, strategiesFilters, strategiesParameters]);
+
+  return (
+    <>
+      <Button onClick={handleOpen} disabled={isLoading} variant="outlined" endIcon={<KeyboardArrowDownIcon />}>
+        <FormattedMessage defaultMessage="Filters" description="all-strategies-table.filters" />
+      </Button>
+      <Popover
```

Comment:
Seems like we overly use this, maybe it would be good to have it componentized with overridable params?

Reply:
Actually this is the second time we are using it. I tried to abstract it but the first implementation has a different button component, so the open/close behaviour should be handled by the consumer of the component, we wouldn't really gain much in my opinion. I have set some styles by default in our theme though! And also remove code from our old footer